<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Qixin He" />


<title>Main</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">var_functionalDiff</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="main.html">Manuscript</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/heqixin">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Main</h1>
<h4 class="author"><em>Qixin He</em></h4>
<h4 class="date"><em>1/2/2019</em></h4>

</div>


<p><strong>Last updated:</strong> 2019-01-08</p>
<strong>workflowr checks:</strong> <small>(Click a bullet for more information)</small>
<ul>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>R Markdown file:</strong> up-to-date </summary></p>
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Environment:</strong> empty </summary></p>
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Seed:</strong> <code>set.seed(20181206)</code> </summary></p>
<p>The command <code>set.seed(20181206)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Session information:</strong> recorded </summary></p>
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Repository version:</strong> <a href="https://github.com/heqixin/var_functionalDiff/tree/ee404445ce326eebb0619ef0d98a658dd0eef2e9" target="_blank">ee40444</a> </summary></p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
<pre><code>
Untracked files:
    Untracked:  output/Fig1_varFunc.ai
    Untracked:  output/Fig2_EIR_transmissionRate_tradeoff.pdf

Unstaged changes:
    Modified:   analysis/_site.yml

</code></pre>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes. </details>
</li>
</ul>
<details> <summary> <small><strong>Expand here to see past versions:</strong></small> </summary>
<ul>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
File
</th>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
<th style="text-align:left;">
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/heqixin/var_functionalDiff/blob/ee404445ce326eebb0619ef0d98a658dd0eef2e9/analysis/main.Rmd" target="_blank">ee40444</a>
</td>
<td style="text-align:left;">
Qixin He
</td>
<td style="text-align:left;">
2019-01-08
</td>
<td style="text-align:left;">
wflow_publish(“analysis/*Rmd“)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/heqixin/var_functionalDiff/eef8f7cec761747ff2326113acf6359c82ef2c93/docs/main.html" target="_blank">eef8f7c</a>
</td>
<td style="text-align:left;">
Qixin He
</td>
<td style="text-align:left;">
2019-01-08
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/heqixin/var_functionalDiff/blob/999c8d04ae1d3e11487ba44c0b6488f9b4ae0a01/analysis/main.Rmd" target="_blank">999c8d0</a>
</td>
<td style="text-align:left;">
Qixin He
</td>
<td style="text-align:left;">
2019-01-08
</td>
<td style="text-align:left;">
wflow_publish(“analysis/*Rmd“)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/heqixin/var_functionalDiff/2ab1b9f248b3a4726eec0023fd9a8cc5549e7c70/docs/main.html" target="_blank">2ab1b9f</a>
</td>
<td style="text-align:left;">
Qixin He
</td>
<td style="text-align:left;">
2019-01-08
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/heqixin/var_functionalDiff/blob/b5ea7da5aa25551c6b711f824bc374e0316a04cd/analysis/main.Rmd" target="_blank">b5ea7da</a>
</td>
<td style="text-align:left;">
Qixin He
</td>
<td style="text-align:left;">
2019-01-08
</td>
<td style="text-align:left;">
wflow_publish(“analysis/*Rmd“)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/heqixin/var_functionalDiff/2c63fd858ed7bd133fa3a17023437ae780500557/docs/main.html" target="_blank">2c63fd8</a>
</td>
<td style="text-align:left;">
Qixin He
</td>
<td style="text-align:left;">
2019-01-08
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/heqixin/var_functionalDiff/08ba1d04f1031ea76c7f3b04daf0571ffc3269d0/docs/main.html" target="_blank">08ba1d0</a>
</td>
<td style="text-align:left;">
Qixin He
</td>
<td style="text-align:left;">
2019-01-08
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/heqixin/var_functionalDiff/blob/82e67a54df4684f790967adb14f1015632f3c42b/analysis/main.Rmd" target="_blank">82e67a5</a>
</td>
<td style="text-align:left;">
Qixin He
</td>
<td style="text-align:left;">
2019-01-08
</td>
<td style="text-align:left;">
wflow_publish(“analysis/*Rmd“)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/heqixin/var_functionalDiff/538ad32a72faadadb8904e35c43b167c3a8e2a3c/docs/main.html" target="_blank">538ad32</a>
</td>
<td style="text-align:left;">
Qixin He
</td>
<td style="text-align:left;">
2019-01-08
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/heqixin/var_functionalDiff/blob/16edf13a8f3d56edaef01315bd4a06c29233301d/analysis/main.Rmd" target="_blank">16edf13</a>
</td>
<td style="text-align:left;">
Qixin He
</td>
<td style="text-align:left;">
2019-01-08
</td>
<td style="text-align:left;">
wflow_publish(“analysis/*Rmd“)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/heqixin/var_functionalDiff/72e7fd9c45306a37c5231df2a7b678f2f90d32c1/docs/main.html" target="_blank">72e7fd9</a>
</td>
<td style="text-align:left;">
Qixin He
</td>
<td style="text-align:left;">
2019-01-08
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/heqixin/var_functionalDiff/blob/f4bd3fa3059f999575c14861381814ff7e8c6654/analysis/main.Rmd" target="_blank">f4bd3fa</a>
</td>
<td style="text-align:left;">
Qixin He
</td>
<td style="text-align:left;">
2019-01-08
</td>
<td style="text-align:left;">
wflow_publish(“analysis/*Rmd“)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/heqixin/var_functionalDiff/69a6edcc3e2832072dce1fcb49491bc56c4a91f8/docs/main.html" target="_blank">69a6edc</a>
</td>
<td style="text-align:left;">
Qixin He
</td>
<td style="text-align:left;">
2019-01-08
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/heqixin/var_functionalDiff/blob/c12e60141f800de97ef1ebcfff4b75b85ddff287/analysis/main.Rmd" target="_blank">c12e601</a>
</td>
<td style="text-align:left;">
Qixin He
</td>
<td style="text-align:left;">
2019-01-08
</td>
<td style="text-align:left;">
wflow_publish(“analysis/*Rmd“)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/heqixin/var_functionalDiff/156ca226ff84ded4ee0b240edfdf028ea01abde3/docs/main.html" target="_blank">156ca22</a>
</td>
<td style="text-align:left;">
Qixin He
</td>
<td style="text-align:left;">
2019-01-08
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/heqixin/var_functionalDiff/blob/e16f8d4bbee693e518b7ea3a3abbe4c87f465918/analysis/main.Rmd" target="_blank">e16f8d4</a>
</td>
<td style="text-align:left;">
Qixin He
</td>
<td style="text-align:left;">
2019-01-08
</td>
<td style="text-align:left;">
wflow_publish(“analysis/*Rmd“)
</td>
</tr>
</tbody>
</table>
</ul>
<p></details></p>
<hr />
<div id="diversification-in-receptors-and-local-density-dependent-selection-could-be-key-to-diverse-functional-group-of-antigens" class="section level1">
<h1>Diversification in receptors and local density-dependent selection could be key to diverse functional group of antigens</h1>
<p>Authors: Qixin He, Mercedes Pascual</p>
<div id="main-text" class="section level2">
<h2>Main text</h2>
<p>This project is to decipher how or what kind of within-host dynamics would influence the evolutionary fate of antigens.</p>
</div>
<div id="abstract" class="section level2">
<h2>Abstract</h2>
<p>In most parasite-host dynamics, evolutionary timescales cannot be separated from ecological timescales. The long-term evolutionary fate of a parasite gene family could be potentially determined by many ecological factors of within-host dynamics. It is usually hard to disentangle these factors (i.e., understand which ones are the cause and which are consequences) because they could lead to similar patterns. Here we investigate the evolution of a gene family that codes the major antigen during blood transmission stage of malaria, the <em>var</em> genes. Three categories of ecological factors influence the fate of a gene: those that determine the parasites’ growth rate, duration of infection, and diversification rates. We found that through many combinations of factors that influence these three traits, evolutionary force consistently selects for high diversity of genes, low overlap between strains and minimal differences in absolute fitnesses of genes. Despite potential tradeoffs could exist to minimize the absolute differences between genes, fast evolutionary rates of genes will result in genes with the highest absolute fitnesses given large effective population size of parasites. More importantly, stable composition of genes of different absolute fitnesses could only exist if they bind to plethora of receptors which will have density-dependent selection among parasites within the same receptors but not between. Therefore, we hypothesize that partition in receptor binding and the existence of human polymorphism in receptors are the key to diversification and stable existence of different ups groups of <em>var</em> genes. Other related features of the genes are the consequences of effective binding to different receptors (and probably how similar and abundant the receptors are to each other). We also found that linear increase in transmission rates result in power-law increase in EIR and the number of new genes, rendering the disease scenarios into different regimes completely, which demonstrate how effective control in transmission rates could have much greater returns. When compared to empirical data, we found that regions with low transmissions have much greater variation in the composition of var ups types than regions with high transmissions. These results demonstrate that under strong selection, evolutionary force prefers diversity only when it’s minimizing the competition along the axis, and temporary coexistence of different strategies could be very unstable. Through our simulation, we also showed that the age of a gene is correlated with its overall functionality, with old genes be much less likely to be dysfunctional. Prevelance of gene ages is a traveling wave: very old genes are less because they have been known by all the hosts, or mutated away, while new genes still require sometime to gain preference.</p>
</div>
<div id="introduction" class="section level2">
<h2>Introduction</h2>
<div id="whats-the-background-and-why-we-study-it" class="section level3">
<h3>what’s the background and why we study it?</h3>
<p>Gene families involved in parasite-host armsrace evolve dynamically under birth-and-death evolution with different selection mechanisms. Relative strengths of birth-death rates and selection dictate the survival and coexistence of member genes. Parasite antigens are particularly uniquely positioned because they are under two types of selection from the host: one exherted from host immune system to escape antibody recognition from the host, one is to maintain the binding functions of the genes towards the receptors of the host membrane binding. These two forces are selecting in opposite directions of diversity: the one from immune system is to diversify the pool of protein diversity as much as possible, especially for the immunodominant regions. Binding functions towards receptors, on the other hand, selects for conservation of the protein structure so that binding is stable as well as the protein structure. Many potential trade-offs to balance cost and benefits of diversification during within-host dynamics explain the existence of a diverse array of parasite infection strategies (Fig. 1a). Lesser known is to what extent different ecological factors in within-host dynamics influence the long term evolutionary trend. In addition, if there are more general trend of what type of ecological factors would be consistently important.</p>
</div>
<div id="specific-function-of-the-genes" class="section level3">
<h3>specific function of the genes</h3>
<p>In <em>falciparum</em> malaria transmission, <em>var</em> gene family, which codes <em>Pf</em>Emp1 on the surface of infected human red blood cells, is not only the major blood-stage antigen to our immune system, but also confer binding to host epithelial cells or other non-infected red blood cells to avoid splenic clearance and enhance reproduction rates of the parasites. There are 50-60 <em>var</em> genes per parasite genome, each generation, the parasite only display one of the phenotypes to the host. Despite overall immense diversity of amino acid composition of var genes within and between parasite genomes, major groups of var genes exist based on their 5’-flanking regions called ups that control the expression of genes .</p>
<p>Different ups types have average difference in many features. Ups types determine {} genes’ propensity to exchange DNA with other {} genes in the genome via ectopic recombination . In turn, differences in recombination rate result in different levels of diversity and conservation among ups types: upsA genes form a separate recombination group which shares few homologous segments with upsB/upsC genes ; they display different characteristics in terms of expression order, binding affinity to host receptor proteins, as well as disease severity . UpsA genes are also preferentially expressed in hosts with limited immunity , exhibiting higher serodominance in infants than other var groups seroprevalent with age .</p>
<p>It is hard to disentangle however, which properties are the causes and which are the consequences. And how the potential ecological differences influence the longterm evoluitonary differences between the groups.</p>
</div>
<div id="possible-hypotheses-that-explain-the-divergence-in-their-properties" class="section level3">
<h3>possible hypotheses that explain the divergence in their properties</h3>
<p>There are different hypotheses regarding the average differences we see in the groups. First, tradeoff between transmission rate and duration of infection. The binding of the gene to epithelial receptors of surfaces determine the growth rate of specific parasites. At the same time, clearance rate of immune responses could be correlated with the abundance of the parasites. Total transmissibility (Fig. 1b) is the product of transmission rate (proportional to growth rate of the parasites) and the duration of infection (negatively correlated with the speed of clearance from immune responses) (Greischar et al. 2016, Klein et al. 2014).</p>
<p>Two transmission strategies coexist: fast recombination at risk of producing non-functional proteins and slow recombination to maintain functions at the risk of immunity gained by hosts (Greischar et al. 2016, Klein et al. 2014).</p>
<p>There’s no tradeoff between transmission rate and possiblity. It’s growth rate difference and the inherent recombination higher. Varying degree of recombination rates difference. But the percentage of genes are free to vary.</p>
<p>Inherent rate decrease. Purifying selection is prevalent. positive selection among different groups.</p>
</div>
<div id="how-we-are-going-about-it-what-questions-are-we-answering" class="section level3">
<h3>how we are going about it? what questions are we answering?</h3>
</div>
</div>
<div id="results" class="section level2">
<h2>Results</h2>
<div id="trade-off-between-transmission-rate-and-duration-of-infection" class="section level3">
<h3>trade-off between transmission rate and duration of infection</h3>
<p>If total transmissibility is kept constant while transmission rate and duration of infection trade off in naive hosts, then percentage of genes with lower transmission rate but higher duration of infection will be favored in higher transmission intensity.</p>
<div class="figure" style="text-align: center">
<img src="./output/Fig2_EIR_transmissionRate_tradeoff.png" alt="genes with longer duration is preferred when transmission intensity is high" width="100%" />
<p class="caption">
genes with longer duration is preferred when transmission intensity is high
</p>
</div>
<div class="figure">
<img src="Fig2_EIR_transmissionRate_tradeoff.png" alt="Fig2" />
<p class="caption">Fig2</p>
</div>
</div>
<div id="selection-minimizes-fitness-differences-among-genes" class="section level3">
<h3>selection minimizes fitness differences among genes</h3>
</div>
<div id="density-dependent-receptor-selection-stablizes-the-functional-group-composition" class="section level3">
<h3>density-dependent receptor selection stablizes the functional group composition</h3>
</div>
<div id="regime-shift-determined-by-effective-population-size" class="section level3">
<h3>regime shift determined by effective population size</h3>
</div>
<div id="gene-age-functionality-and-immunity-niche-distribution" class="section level3">
<h3>gene age, functionality and immunity niche distribution</h3>
</div>
</div>
<div id="discussion" class="section level2">
<h2>Discussion</h2>
<div id="summary-of-findings" class="section level3">
<h3>summary of findings</h3>
</div>
<div id="significance-of-these-findings-on-our-understanding-of-evolutionary-strategies-or-parasite-host-arms-race" class="section level3">
<h3>significance of these findings on our understanding of evolutionary strategies?? or parasite-host arms race</h3>
</div>
<div id="indication-of-the-transmission-difference-regimes" class="section level3">
<h3>indication of the transmission difference regimes</h3>
</div>
<div id="a-general-picture-of-age-of-genes-and-their-functions-a-traveling-wave" class="section level3">
<h3>a general picture of age of genes and their functions, a traveling wave</h3>
</div>
<div id="chessons-theory-of-co-existence-under-the-context-of-co-evolution-and-constant-innovation" class="section level3">
<h3>Chesson’s theory of co-existence under the context of co-evolution and constant innovation</h3>
</div>
</div>
<div id="methods" class="section level2">
<h2>Methods</h2>
<div id="basic-transmission-model-description" class="section level3">
<h3>basic transmission model description</h3>
</div>
<div id="within-host-dynamics-options" class="section level3">
<h3>within-host dynamics options</h3>
<div id="trade-off-between-transmission-rate-and-duration-of-infection-1" class="section level4">
<h4>trade-off between transmission rate and duration of infection</h4>
</div>
<div id="how-co-infection-influences-transmission" class="section level4">
<h4>how co-infection influences transmission</h4>
</div>
<div id="recombination-rates-in-different-groups" class="section level4">
<h4>recombination rates in different groups</h4>
</div>
</div>
<div id="data-availability-reproducibility" class="section level3">
<h3>data availability, reproducibility</h3>
</div>
</div>
<div id="acknowledgements" class="section level2">
<h2>Acknowledgements</h2>
</div>
<div id="session-information" class="section level2">
<h2>Session information</h2>
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.3.2 (2016-10-31)
Platform: x86_64-apple-darwin13.4.0 (64-bit)
Running under: macOS  10.14

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
 [1] workflowr_1.1.1   Rcpp_0.12.13      digest_0.6.12    
 [4] rprojroot_1.3-2   R.methodsS3_1.7.1 backports_1.1.2  
 [7] git2r_0.22.1      magrittr_1.5      evaluate_0.10.1  
[10] highr_0.7         stringi_1.1.5     whisker_0.3-2    
[13] R.oo_1.21.0       R.utils_2.5.0     rmarkdown_1.10   
[16] tools_3.3.2       stringr_1.2.0     yaml_2.1.19      
[19] htmltools_0.3.6   knitr_1.20       </code></pre>
</div>
</div>

<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>

<hr>
<p>
  This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
  analysis was created with
  <a href="https://github.com/jdblischak/workflowr">workflowr</a> 1.1.1
</p>
<hr>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
